Makefile

```diff
@@ -404,7 +404,7 @@ test-sim-fullappsimulation:
 	$(call run-sim-test,"TestFullAppSimulation",TestFullAppSimulation,100,200,30m)
 
 test-sim-import-export:
-	$(call run-sim-test,"test-import-export",TestAppImportExport,100,200,30m)
+	$(call run-sim-test,"test-import-export",TestAppImportExport,50,100,30m)
```

Comment:
I think its okay to reduce the number of transactions in a block for this test .
This test compares stores beween two copies of the application , ( compares every key and value ) . The higher number of trasnactions leads to higher number of keys , which sometimes cause the checking function to fail .